example/feature_demo/demo_service.pb.gorm.go

```diff
@@ -302,3 +307,130 @@ func (m *IntPointDefaultServer) CreateSomething(ctx context.Context, in *Somethi
 	}
 	return &Something{}, nil
 }
+
+type IntPointTxnDefaultServer struct {
+}
+type IntPointTxnCreateCustomHandler interface {
+	CustomCreate(context.Context, *CreateIntPointRequest) (*CreateIntPointResponse, error)
+}
+
+// Create ...
+func (m *IntPointTxnDefaultServer) Create(ctx context.Context, in *CreateIntPointRequest) (*CreateIntPointResponse, error) {
+	if custom, ok := interface{}(m).(IntPointTxnCreateCustomHandler); ok {
+		return custom.CustomCreate(ctx, in)
+	}
+	txn, ok := tkgorm.FromContext(ctx)
+	if !ok {
+		return &CreateIntPointResponse{}, errors.New("Database Transaction For Request Missing")
+	}
+	db := txn.Begin()
```

Comment:
I mimicked the behavior in the [Toolkit Readme ](https://github.com/infobloxopen/atlas-app-toolkit#middlewares). Did I miss anything important? It's hard to apply it to the example app without merging it so the gentool can grab it first.

Reply:
I can tell you how I personally test my local changes to protoc-gen-gorm:
- Push your changes to your protoc-gen-gorm fork, e.g. `https://github.com/calebjh/protoc-gen-gorm`
- Clone the gentool
- Modify gentool's glide.yaml so it grabs protoc-gen-gorm from your fork:
```
  # gorm code generator
  - package: github.com/infobloxopen/protoc-gen-gorm
    repo: https://github.com/calebjh/protoc-gen-gorm
```
- Build gentool with the tag which is required by contacts-app(v3 now), modify it's makefile accordingly
- Generate contacts-app code
I personally don't feel any burden when doing this..

Reply:
That works great! I didn't know about the `repo` option in glide. Thanks!